### ISSUE TITLE
elm-0.18: Content-Type is set twice for JSON

### ISSUE THREAD

**Issue by jb55** [mattjbray/servant-elm#19]
It looks like `Http` already adds `Content-Type: application/json` when using jsonBody:

https://github.com/elm-lang/http/blob/99c00a2dac21dda1d4954515c79e6f28e431573f/src/Http.elm#L287-L292

The generated code also adds it, causing application/json to appear twice in the `Content-Type` header. This causes servant to reject the request with `415: Unsupported Media Type`

My current workaround:

```sh
find frontend/src/Generated -name '*.elm' -exec \
  sed -i 's/Http.header "Content-Type" "application\/json"//' {} \;
```

**Comment by jb55** [comment]
Fixed in d758f93

**Comment by kgashok** [comment]
I'm a noob to Elm and HTTP API calls. This problem had me going in circles for almost ~2 hours until I found this issue. Some sample code would be much appreciated.
